modules/core/nuxeo-platform-commandline-executor/src/main/java/org/nuxeo/ecm/platform/commandline/executor/service/executors/ShellExecutor.java

```diff
@@ -66,6 +67,14 @@
     /** Used to split the contributed command, NOT the passed parameter values. */
     protected static final Pattern COMMAND_SPLIT = Pattern.compile("\"([^\"]*)\"|'([^']*)'|[^\\s]+");
 
+    protected final boolean useTimeout;
+
+    public static final int DEFAULT_TIMEOUT_S = 86400; // 24h
```

Comment:
Please use `Duration.ofDays(1).toSeconds()` (or `getSeconds` if needed for backporting) to express the value, that's been our convention for all recent code.

Reply:
and I have to add a cast this to int? 24*3600 sounds better.

Reply:
It's all about readability and for the reader not having any doubt that the code is correct.